sys/net/gnrc/netif/gnrc_netif_device_type.c

```diff
@@ -90,9 +90,12 @@ int gnrc_netif_eui64_from_addr(const gnrc_netif_t *netif,
                 }
 #endif  /* defined(MODULE_NETDEV_IEEE802154) || defined(MODULE_XBEE) */
                 /* Intentionally falls through */
-            default:
-                return l2util_eui64_from_addr(netif->device_type, addr,
-                                              addr_len, eui64);
+            default: {
+                int res = l2util_eui64_from_addr(netif->device_type, addr,
+                                                 addr_len, eui64);
+                assert(res >= 0);
```

Comment:
```suggestion
                assert(res != -ENOTSUP);
```

`-EINVAL` is should IMHO still be reported to the caller

Reply:
fine with me